 
<launch>

 <node pkg="using_markers" type="calculation_for_goal1" name="" args="$(find using_markers)/src/calculation_for_goal1" output="screen">


      <param name="vp_number" value="3.0" />
      <param name="vp_width" value="2.1" />
      <param name="vp_depth" value="3.26" />
      <param name="offset_x" value="1.05" />      <!--vp_width/2-->
      <param name="offset_y" value="2.343" />     <!--vp_.depth+car_length)/2-0.49 -->    
      <param name="vp_line_width" value="0.1" />   
      <param name="vp_origin_x" value="0.0" />
      <param name="vp_origin_y" value="0.0" />
      <param name="vp_yaw" value="2.45" />           <!-- between 0 and 1PI-->  
         
      <param name="pp_number" value="3.0" />
      <param name=" pp_width" value="3.51" />
      <param name="pffset_x" value="0.80" />         <!--pp_.depth-car_length)/2+0.49-0.2 -->    
      <param name="pffset_y" value="1.075" />         <!--carpark_vertical.depth/2 -->  
      <param name=" pp_depth" value="2.15" />
      <param name="pp_line_width" value="0.1" />
      <param name="pp_origin_x" value="-5.0" />
      <param name="pp_origin_y" value="7.5" />
      <param name="pp_yaw" value="0.0" />


      <param name="p_number" value="3.0" />
      <param name="p_width" value="2.1" />
      <param name="p_depth" value="3.26" />
      <param name="ffset_x" value="1.05" />      <!--vp_width/2-->
      <param name="ffset_y" value="0.49" />     <!--vp_.depth+car_length)/2-0.49 -->    
      <param name="p_line_width" value="0.1" />   
      <param name="p_origin_x" value="-14.84" />
      <param name="p_origin_y" value="16.6" />
      <param name="p_yaw" value="0.785" />           <!-- between 0 and 1PI-->  



      <param name="radius" value="2.60" />
      <param name="angle_max" value="1.57" />
      <param name="angle_d" value="0.1" />
      <param name="distance_between_two_centers" value="0.5" />
      <param name="Cicle_numbers" value="20" />     
      <param name="lot_number" value="1" /> 
        
      </node>


<node name="rviz" pkg="rviz" type="rviz" args="$(find using_markers)/rviz/calculation_for_goal1.rviz"/>


</launch>




